static/js/components/VideoPlayer_test.js

```diff
@@ -40,18 +41,23 @@ describe('VideoPlayer', (props = {}) => {
     sandbox = sinon.sandbox.create();
     videojsStub = sandbox.stub(libVideo, 'videojs');
     cornerFunction = sandbox.stub();
+    gaStub = sandbox.stub(libGA, 'sendGAEvent');
     playerStub = {
       el_: {
-        style: {}
+        style: {},
+        dispatchEvent: sandbox.stub()
       },
       tracks: [],
+      on: sandbox.stub(),
+      currentTime: function() {return 30.5;},
```

Comment:
Can you change this function and `duration` to be arrow functions: `() => 30.5` for example? (The other functions were updated in a separate PR)

Reply:
OK